app/controllers/omniauth_callbacks_controller.rb

```diff
@@ -2,8 +2,35 @@ class OmniauthCallbacksController < ApplicationController
   def callback
     auth = request.env["omniauth.auth"]
 
-    session[:user_info] = auth.extra.raw_info
+    session[:user_info] = if Rails.env.development?
+      {
+        "trn" => "1886088",
+        "family_name" => "John",
+        "given_name" => "Smith",
+        "birthdate" => "1990-11-09",
+        "ni_number" => "TK519024B",
+        "trn_match_ni_number" => "true",
+        "email" => "john.smith@example.com",
+        "phone_number" => "01234567890"
+      }
+    else
+      auth.extra.raw_info
+    end
 
-    redirect_to claim_path(policy: "additional-payments", slug: "teacher-detail")
+    redirect_to claim_path(policy: policy.routing_name, slug: "teacher-detail")
+  end
+
+  private
+
+  def claim_id
+    @claim_id = session["claim_id"]
+  end
+
+  def current_claim
+    @current_claim ||= Claim.find_by(id: claim_id)
+  end
+
+  def policy
+    @policy ||= current_claim.policy
```

Comment:
Consider using `PartOfClaimJourney::current_claim` as there is a bit of logic needed around retrieving this depending on which journey and where they are in the journey.

Reply:
Thanks for that extra context. It makes sense. The change broke 12 specs so I'm going to work through those now.